src/main/java/net/coru/kloadgen/randomtool/util/ValidType.java

```diff
@@ -0,0 +1,85 @@
+/*
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  * License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package net.coru.kloadgen.randomtool.util;
+
+import java.util.Set;
+
+public class ValidType {
```

Comment:
This looks like an Enumeration right?

Reply:
At the moment I put these as a constants file, but in the future could be converted to a enum without a doubt or several. Dividing for each time of thing (map, list, value)